assets/src/block-editor/components/sidebar-notification/index.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
```

Comment:
Following comment from above.

```suggestion
import PropTypes from 'prop-types';
import { ReactElement } from 'react';
```

Reply:
Addressed in efeba44.